torch/lib/build_libs.sh

```diff
@@ -142,6 +142,7 @@ function build_aten() {
   cd  build/aten
   ${CMAKE_VERSION} ../../../../aten \
   -DCMAKE_BUILD_TYPE=Release \
+  -DNO_CUDA=$((1-$WITH_CUDA)) \
```

Comment:
A bit skeptical.

Reply:
That doesn't work. We e.g. support "YES" or "Y" for WITH_CUDA

Reply:
I copy pasted this from somewhere else in the build :) so it has been this way for a while. We should probably figure out what it actually has to be...